### PR TITLE
Project components

### PR DESCRIPTION
**Pull request by Gopxfs**
# :ledger: Summary
Add basic components & page skeleton to the project.

### AddNewBook.js :link:
A form that can add a new book by adding a title and an author.
### Book.js :link:
Creates a single book by receiving a title and an author.
### Books.js :link:
Displays the list of books.
### Categories.js :link:
Displays a "Check status" button.
### NavBar.js :link:
The nav containing NavLinks to the routes.

### Books.js
The main page. Displays the book list and the add new book form.
### Categories.js
Displays the categories page.

### CODE REVIEW

**Review by Mre55 — COMMENTED** [review]
**Hi @Gopxfs,**

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :negative_squared_cross_mark:

You have submitted a project, that linter is not passing.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: src/components/AddNewBook.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const AddNewBook = () => (
+  <div className="AddNewBook">
+    <input type="text" placeholder="Book title" />
+    <input type="text" placeholder="Author" />
+  </div>
```
Comment (mikethreels) [inline src/components/AddNewBook.js]: - [x] the form should have a submit button as well and it should also be wrapped in a form tag for it to work.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Gabriel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @Gopxfs 👋, 

Well done implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights 🎯 
✔️ Descriptive PR
✔️ Meaningful commit messages
✔️ Linters passing

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**